lrdd/row.go

```diff
@@ -0,0 +1,52 @@
+package lrdd
+
+import "fmt"
+
+type Row struct {
+	Key   string
+	Value MarshalUnmarshaler
+}
+
+type MarshalUnmarshaler interface {
+	MarshalMsg([]byte) ([]byte, error)
+	UnmarshalMsg([]byte) ([]byte, error)
+	ID() RowID
+	String() string
+}
+
+type RowID int32
```

Comment:
`RowID` represents the type of row, not an identifier of each row itself, so I think `RowType` is more intuitive.

Reply:
Done. 468c7ec22ea0f4d25e9ae093e441a53c8532350b